lambda-buffers-codegen/test/Test/LambdaBuffers/Codegen/Rust.hs

```diff
@@ -0,0 +1,29 @@
+module Test.LambdaBuffers.Codegen.Rust (tests) where
+
+import Data.Aeson qualified as A
+import Data.Functor (void)
+import LambdaBuffers.Codegen.Rust.Config qualified as R
+import Paths_lambda_buffers_codegen qualified as Paths
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.HUnit (testCase)
+
+tests :: TestTree
+tests =
+  testGroup
+    "LambdaBuffers.Codegen.Rust"
+    [ configParses
+    ]
```

Comment:
why don't you add tests like Plutarch.tests?

Reply:
I'll come back to this, once I finished with the testsuites